modules/agent-installer-configuring-fips-compliance.adoc

```diff
@@ -10,6 +10,11 @@
 
 During a cluster deployment, the Federal Information Processing Standards (FIPS) change is applied when the Red Hat Enterprise Linux CoreOS (RHCOS) machines are deployed in your cluster. For Red Hat Enterprise Linux (RHEL) machines, you must enable FIPS mode when you install the operating system on the machines that you plan to use as worker machines.
 
+[IMPORTANT]
+====
+{product-title} requires the use of a FIPS-capable installation binary to install a cluster in FIPS mode.
```

Comment:
```suggestion
You must use a FIPS-capable installation binary for {product-title} to install a cluster in FIPS mode.
```

- Consider rephrasing in a more user-focused approach -- something like the above suggested change...